lib/Text_Wiki/Text/Wiki/Parse/Default/Email.php

```diff
@@ -40,20 +40,25 @@ class Text_Wiki_Parse_Email extends Text_Wiki_Parse {
     *
     */
 
-	public $regex = '[_a-z0-9\-]+(?:\.[_a-z0-9\-]+)*@[a-z0-9\-]+(?:\.[a-z0-9\-]+)+';
+    public $regex = '
```

Comment:
Long term we'll have to do something else with this parser, probably just use PHP's `filter_var` function and return matches that way, but that's out of scope for this PR.